### ISSUE TITLE
A question about the large performance gap compared with ULIP

### ISSUE THREAD

**Issue by chenxinli001** [deeptibhegde/CLIP-goes-3D#16]
I am so curious about CG3D and a co-current work called ULIP. It seems that there is no significant difference between the methodology design of two works. But why the performance  of CG3D is much lower than ULIP (please see the attached fig). For example, the zero-shot ACC on ScanObjectNN via PointMLP:  CG3D vs. ULIP = 25.0 vs. 44.6. Any reasonable insights for this perfomance gap?

![image](https://github.com/deeptibhegde/CLIP-goes-3D/assets/58136000/38aa7187-8726-4c46-a33d-4c7047382b2b)

**Comment by deeptibhegde** [comment]
We suspect that the ULIP method leverages some data augmentation techniques that improve the zero-shot performance of the network, particularly in cases where the test set is very different than the pre-training dataset (as in the case of ScanObjectNN). We observed that the data augmentation strategy makes a large difference on final zero-shot performance in our experiments. We hope to improve on this and potenially boost our results for the next version of the codebase.

**Comment by chenxinli001** [comment]
Thank you for your response and insightful comments!
